ntbs-service/Models/SearchParameters.cs

```diff
@@ -6,6 +6,7 @@
 namespace ntbs_service.Models
 {
     [NotMapped]
+    [AtLeastOneProperty("IdFilter", "FamilyName", ErrorMessage = ValidationMessages.SupplyARequiredParam)]

```

Comment:
This will need to be added to for the other 2 required parameters - another way to do this would be to create a new class within searchParameters of required search parameters and map all of them to this kind of attribute - that feels worse to me though

Reply:
This seems fine to me!